### PR TITLE
Create profile views with appropriate conditional rendering, fixed css on certain pages

### PR DESCRIPTION
**Pull request by tiffanyyv**
LoginPage
ProfileView
DateTimePicker Spacing
CalendarMargin

- Create profile views and displays appropriate components depending on whether the user is a teacher or student
- Fixed right margin on calendar view. Fixed/added divs in my calendar page
- Fixed spacing between date/time picker entries on my courses page
- Fixed login button colors to match theme

### CODE REVIEW

**Comment by tiffanyyv** [comment]
@cmjosephs Kevin and I included a media query for the profile view but feel free to suggest different max-widths

File: pages/app/[teacherprofile].js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
```
Comment (cmjosephs) [inline pages/app/[teacherprofile].js]: Nit: useEffect is being imported here even though it hasn't been implemented yet.
I know you guys talked about it but I think I think adding the dynamic route [teacherprofile] would be better in its own directory like how you have student. Just a little clearer what's going on

File: pages/app/[teacherprofile].js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          <p>Description</p>
+        </div>
+      </div>
+        {userType === 'mentor' && <div>Recommended: {currentTeacherRecommend}</div>}
+        {userType === 'student' &&
+          <div>
+            <div>Would you recommend this teacher?</div>
+            <span onClick={handleUpvoteTeacher}>Yes</span>
+            <span>{" "}|{" "}</span>
+            <span onClick={handleDownvoteStudent}>No</span>
```
Comment (cmjosephs) [inline pages/app/[teacherprofile].js:45]: handleDownVoteStudent is not defined. Did you mean to use handleDownVoteTeacher?

File: pages/app/[teacherprofile].js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+export default function TeacherProfile(props) {
+  const [userType, setUserType] = useState('mentor');
+  const [currentTeacherRecommend, setCurrentTeacherRecommend] = useState(10);
+
+  // increment with axios put request
+  // re render page with new count
+  // add user type: teacher or student
+  const handleUpvoteTeacher = () => {
+    setCurrentTeacherRecommend(currentTeacherRecommend++);
```
Comment (cmjosephs) [inline pages/app/[teacherprofile].js:16]: Should be setCurrentTeacherRecommend(currentTeacherRecommend + 1). We are not incrementing the current state, we are adding 1 to the value of the current state and setting a brand new state. Same for downVote function

File: pages/app/[teacherprofile].js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  const handleDownvoteTeacher = () => {
+    setCurrentTeacherRecommend(currentTeacherRecommend--);
```
Comment (cmjosephs) [inline pages/app/[teacherprofile].js:20]: See comment above for handleUpvote

File: pages/app/[teacherprofile].js (hunk 3 above)
Comment (cmjosephs) [reply pages/app/[teacherprofile].js:16]: See how they increment the counter's state in react docs 
https://reactjs.org/docs/hooks-state.html

File: pages/app/calendar.js [hunk 5]
```diff
@@ -26,44 +26,46 @@ import { data } from '../../components/Calendar/data/dummyData.js';
...
+    setAppointmentData(data);
   }, []);
 
   return (
-    <>
-      <Head>
+    <div className="pageData">
+      <div className={styles.calendarContainer}>
+        <Head>
           <title>{`${username}'s Calendar`}</title>
```
Comment (cmjosephs) [inline pages/app/calendar.js:43]: @gunpowder66 @changerbang  Please review this file to make sure everything looks good.

File: pages/app/index.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+// Dashboard
```
Comment (cmjosephs) [inline pages/app/index.js:1]: Remove this file. I think we decided on no dashboard page

File: pages/app/my-profile.js [hunk 7]
```diff
@@ -1,9 +1,30 @@
...
   return (
-    <div className='pageData'>My Profile</div>
+    <div className="pageData">
+      <div className={styles.profileContainer}>
+          <div className={styles.profilePic}>
+            <Avatar
+              className="my-profile-view-avatar"
+              alt="My Profile Picture"
+              src={defaultProfilePic}
+              sx={{ width: 300, height: 300 }}
```
Comment (cmjosephs) [inline pages/app/my-profile.js:19]: I'm thinking we add default dimensions for things like profile pics to constants file. Lmk your thoughts on this

File: pages/app/student-profile/[studentprofile].js [hunk 8]
```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
```
Comment (cmjosephs) [inline pages/app/student-profile/[studentprofile].js:1]: Nit: same as teacherProfile, useEffect is imported but not used yet. Not relevant right now but be aware that we will be trying as much as possible to use SSG and SSR so useEffect will be used sparingly for files under /pages directory

File: pages/app/student-profile/[studentprofile].js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+  return (
+    <div className="pageData">
+      <div className={styles.profileContainer}>
+          <div className={styles.profilePic}>
+            <Avatar
+              className="my-profile-view-avatar"
+              alt="Student Profile Picture"
+              src={defaultProfilePic}
+              sx={{ width: 300, height: 300 }}
```
Comment (cmjosephs) [inline pages/app/student-profile/[studentprofile].js:31]: Same as other profile pages. Lmk your thoughts on default profile pic dims in the constants folder

File: pages/app/student-profile/[studentprofile].js [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+          <h3>Location</h3>
+          <p>Description</p>
+        </div>
+      </div>
+        {userType === 'mentor' &&
+          <div>
+            <div>Recommended: {currentStudentRecommend}</div>
+            <div>Would you recommend this student?</div>
+            <span onClick={handleUpvoteStudent}>Yes</span>
+            <span>{" "}|{" "}</span>
```
Comment (cmjosephs) [inline pages/app/student-profile/[studentprofile].js:44]: Comment explaining what this line does would be helpful. I think I get it but it looks a little odd

**Review by cmjosephs — CHANGES_REQUESTED** [review]
A few comments in here. DM me if for any questions/clarifications

File: pages/app/calendar.js (hunk 5 above)
Comment (changerbang) [reply pages/app/calendar.js:43]: I believe Tiff just added line 40 and 41 "pageData" and "styles.caleendarContainer". If thats just for styling, should be fine.

Comment (mtchang25) [reply pages/app/calendar.js:43]: I believe Tiff just added line 40 and 41 "pageData" and "styles.caleendarContainer". If thats just for styling, should be fine.

File: pages/app/[teacherprofile].js (hunk 1 above)
Comment (tiffanyyv) [reply pages/app/[teacherprofile].js]: Ty for catching this!

File: pages/app/[teacherprofile].js (hunk 3 above)
Comment (tiffanyyv) [reply pages/app/[teacherprofile].js:16]: We put this for now but I think we're going to have to refactor this when we get the backend integrated

**Pull request closed without merging**